palantirnet/the-build
# Download one Acquia backup database at a time.

See #103.

This removes the old Acquia database downloading targets in favor of a new task that downloads new backups selectively via the Acquia Cloud API.

To test:
* Download your Acquia Cloud API credentials and move them to `~/.acquia/cloudapi.conf` on your Vagrant box
* Configure your Acquia site properties in `.the-build/build.default.properties.yml`:
   ```
   acquia:
     site: mysite
     realm: devcloud
   ```
* Run `phing acquia-get-backup` - the latest backup should be downloaded to `artifacts/backups`
* Run `phing acquia-load-backup` - the latest existing backup in `artifacts/backups` will be loaded in to the site db
